src/main/webapp/app/lecture/lecture-detail.component.ts

```diff
@@ -65,4 +75,12 @@ export class LectureDetailComponent implements OnInit {
             ];
         }
     }
+    /**
+     * Trigger the Ingeston of this Lecture in Iris.
+     */
+    ingestLectureInPyris() {
+        this.lectureService.ingestLecturesInPyris(this.lecture.course!.id!, this.lecture.id).subscribe({
+            error: (error) => console.error(`Failed to send Ingestion request`, error),
```

Comment:
This is not a formatted String, so you can simply use single apostrophes here. 
```suggestion
            error: (error) => console.error('Failed to send Ingestion request', error),
```